app/src/main/res/values-es/strings.xml

```diff
@@ -188,6 +188,11 @@
     <string name="settings_record_name_formatting_2_display">estación_fecha_\u200bhora</string>
     <string name="settings_record_name_formatting_3_display">índices_estación_\u200bfecha</string>
 
+    <string name="settings_record_name_formatting_default_display">estación_artista_pista_\u200bfecha_\u200bhora</string>
+    <string name="settings_record_name_formatting_1_display">estación_artista_\u200bpista</string>
+    <string name="settings_record_name_formatting_2_display">estación_fecha_\u200bhora</string>
+    <string name="settings_record_name_formatting_3_display">índice_estación_\u200bfecha</string>
+
```

Comment:
CI is failing because these strings are already in the translation, you should change existing ones instead of duplicating them.